docs/modules/postgresql.md

```diff
@@ -68,12 +68,51 @@ name.
 
 | Parameter Name | Type | Description | Required | Default Value |
 |----------------|------|-------------|----------|---------------|
-| `cores` | `integer` | Specifies vCores, which represent the logical CPU. Valid values are 1 or 2 | N | 1 |
+| `cores` | `integer` | Specifies vCores, which represent the logical CPU. Valid values are 2, 4, 8, 16 | N | 2 |
 | `storage` | `integer` | Specifies the amount of storage to allocate in GB. Ranges from 5 to 1048 | N | 10 |
 | `backupRetention` | `integer` | Specifies the number of days to retain backups. Ranges from 7 to 35 | N | 7 |
 | `backupRedundancy` | `string` | Specifies the backup redundancy, either `local` or `geo` | N | `local` |
 
 
+##### Update
+
+Updates a previously provisioned PostgreSQL DBMS. Currently updating the database extensions is not supported.
```

Comment:
Since this refers to the all-in-one plans and we _do_ have some DBMS-only plans, I think we need to be more specific here than just "previously provisioned DBMS." I don't have any helpful suggestions on how to phrase this, however.